notebooks/1.2-pyc-eda-mix.py

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # EDA MIX data
+# 
+# To explore all the 81 columns of this data frame the NEO team decide to split the EDA into 3 notebooks where:
+# 1- EDA of Structural Data
+# 2- EDA of Rooms data
+# 3- EDA of Mix Data (data that's not about Structural and Rooms data)
+# 
+# This notebook had the role to analyze the MIX data, **focus on the distribution of the variable, and try to get some useful insights**.
+
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+import plotly.express as px
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+#Cdf function
+def ecdf(x):
+    x = np.sort(x)
+    def result(v):
+        return np.searchsorted(x, v, side='right') / x.size
+    return result
+
+def multiple_ecdf(column, target='SalePrice'):
```

Comment:
Some comments will help to understand better the script

Reply:
Okk!